src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-elasticsearch-monitoring-integration.mdx

```diff
@@ -372,7 +372,7 @@ The Elasticsearch integration collects these metrics for clusters:
       </td>
 
       <td>
-        The remaining CPU credits available for dedicated master nodes in the cluster. A CPU credit provides the performance of a full CPU core for one minute. This metric is available only for the t2.micro.elasticsearch, t2.small.elasticsearch, and t2.medium.elasticsearch instance types.
+        The remaining CPU credits available for dedicated master nodes in the cluster. A CPU credit provides the performance of a full CPU core for one minute. This metric is available only for the t2.micro.search, t2.small.search, and t2.medium.search instance types.
```

Comment:
The same comment as before with: This metric is available only for the t2.micro.search, t2.small.search, and t2.medium.search instance types.